tests/js/spec/components/group/suggestedOwners.spec.jsx

```diff
@@ -65,14 +66,18 @@ describe('SuggestedOwners', function () {
     });
 
     const wrapper = mountWithTheme(
-      <SuggestedOwners project={project} group={group} event={event} />,
+      <CommittersProvider>
+        <SuggestedOwners project={project} group={group} event={event} />
+      </CommittersProvider>,
       routerContext
     );
 
-    await tick();
-    await tick(); // Run Store.load and fire Action.loadSuccess
-    await tick(); // Run Store.loadSuccess
-    wrapper.update();
+    await act(async () => {
+      await tick();
+      await tick(); // Run Store.load and fire Action.loadSuccess
```

Comment:
🥲

Reply:
Yeah, we need to just migrate these to rtl, but that's for a separate time